emukit/examples/profet/meta_benchmarks/architecture.py

```diff
@@ -0,0 +1,60 @@
+import torch
+import torch.nn as nn
+from pybnn.util.layers import AppendLayer
+
+
+def get_default_architecture_classification(input_dimensionality: int) -> torch.nn.Module:
+    class Architecture(nn.Module):
```

Comment:
There is a lot of duplication across these methods, to the point that the latter two are identical. This code asks for better reuse

Reply:
yes that makes sense. The current methods are more artifacts of developing

Reply:
so what's the plan regarding these?

Reply:
I mean, most of the code building various architecture still vastly repeats itself.

Reply:
agreed, I refactor the code to just a single function